### DISCUSSION TITLE
Im not able to load university documentation for acreditation on Github education

### DISCUSSION THREAD

**Discussion by Orecul78** [community/community#141848] [GitHub Education]
### Hi everyone,

Hi
Im currently doing a data science degree on UTEL universidad that is an online university, I have tried two times to request this,  I have loaded a photo of dated transcrips and also the same transscripts say is an online unversity so I load them so say that im on a online university.

But still i got rejected can someone please guide me on how to apply? or load a document?

**Comment by cmontano7** [comment]
Just commenting on here to see if/when they answer....I'm experiencing the same. I tried to contact the Helpdesk but then was notified that they are no longer willing to manually approve applications which means that they're using some whacked-out automated system that is clearly not working. I don't know why the only option is to take a photo instead of uploading a file, like normal.

**Reply by dioneja8** [reply]
I'm experiencing the same issue - I don't have hard copies of my study documentation, therefore I thought I could simply upload them. Will it work if I print them out and then take a picture? Seems a bit backwards though.

**Comment by cmkaya** [comment]
Hello,

I have experienced the same problem. It is strange we cannot upload the document. When I try to take a photo, it doesn't appear clearly. In conclusion, the result is rejected.
